index.js

```diff
@@ -1 +1,100 @@
 // implement your API here
+const express = require('express');
+
+const db = require('./data/db');
+
+const server = express();
+
+server.use(express.json());
+
+server.get('/', (req, res) => {
+  res.send('API is working')
+})
+
+// create a user
+server.post('/api/users', (req, res) => {
+  const { name, bio } = req.body;
```

Comment:
I like the use of destructuring to keep the code more concise and also created a new user object instead of using the req.body